src/utils/utils.ts

```diff
@@ -169,9 +168,10 @@ export const importAllFilesFromFolder = (r: any) => {
 export const fetchJson = async (
   url: string | URL,
   method: string = "GET",
-  payload?: any
+  payload?: any,
+  additionalHeaders?: HeadersInit,
```

Comment:
Since it is used to update headers, not initialize them, and we do not support arrays of 2-tuples, I would use `Record<string, string>` here instead of `HeadersInit`.

Reply:
Done